### DISCUSSION TITLE
功能投票中的功能有一个预计的发布时间吗（

### DISCUSSION THREAD

**Discussion by xxtg666** [Meloong-Git/PCL#3468] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/68174188/e741feb6-b2fd-4d6a-bdc1-61e71f118538)

已经有9个功能超过1000票了，第一个甚至快到2000了
想问问这些功能的开发进度怎么样了（

**Comment by LTCatt** [comment]
因为票数更高的功能已经改成处理中了

**Reply by SqrtTan** [reply]
6

**Reply by MoYuan-CN** [reply]
啊这...

**Reply by iPodtouch6557** [reply]
为啥要这么表态啊，~不是很对吗咕~
